src/app/core/services/authentication/authentication.service.ts

```diff
@@ -9,6 +9,7 @@ import { ApiService } from '../api/api.service'
 const routes = {
   login: '/users/authenticate',
   logout: '/users/logout',
+  signup: '/users/register'
```

Comment:
@mwd-luongmai are we using `/user` or `/users`? The previous version used users, but I know in the chat you said `user`

Reply:
hi @1ceb3rg, this is the api route , not Angular route. Hope you don't confuse about that